roles/confluent.ksql/tasks/rbac.yml

```diff
@@ -79,7 +79,7 @@
         "resourcePatterns": [
           {
             "resourceType":"Topic",
-            "name":"{{ksql_final_properties['ksql.service.id']}}-ksql_processing_log",
+            "name":"{{ksql_final_properties['ksql.service.id']}}-ksql_processing_log", // TODO is this needed?
```

Comment:
Are you saying this is a duplicate of the rolebinding below? It appears that way. Is the exact topic name in our docs?

My inclination is to remove this as well. WDYT?

Reply:
Yes, agree. I will remove it. I think it tries to provide the same permissions from before we support configuring processing log here.

Now that we set `ksql.logging.processing.topic.name`, this PR should fix the permissions to work properly when `ksql_processing_log` changes.